tests/test_notifications.py

```diff
@@ -33,6 +33,12 @@
 
 class TestNotificationsModels(OsfTestCase):
 
+    @classmethod
+    def setUpClass(cls):
```

Comment:
If you've got a bunch of different test classes that need to set this setting to `True` (especially since they're all in the same file), you might want to make a wrapper class for `OsfTestCase` that just does does this step. Something like
```python
class NotificationTestCase(OsfTestCase):
    @classmethod
    def setUpClass(cls):
        pass  # modify setting

    @classmethod
    def tearDownClass(cls):
       pass  # unmodify setting

class TestNotificationsModels(NotificationTestCase):
    # setUp, tests, etc.

class TestSubscriptionView(NotificationTestCase):
    # ...
```